<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hello QTPY</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QPlainTextEdit" name="plainTextEdit"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>파일</string>
    </property>
    <addaction name="actionload"/>
    <addaction name="actionsave"/>
    <addaction name="actionsave_as"/>
    <addaction name="actionclose"/>
   </widget>
   <widget class="QMenu" name="menu_edit">
    <property name="title">
     <string>편집</string>
    </property>
    <addaction name="actionundo"/>
    <addaction name="actioncut"/>
    <addaction name="actioncopy"/>
    <addaction name="actionpaste"/>
    <addaction name="separator"/>
    <addaction name="actionfind"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_edit"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionsave">
   <property name="text">
    <string>save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionload">
   <property name="text">
    <string>load</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionsave_as">
   <property name="text">
    <string>save as</string>
   </property>
  </action>
  <action name="actionclose">
   <property name="text">
    <string>close</string>
   </property>
  </action>
  <action name="actionundo">
   <property name="text">
    <string>undo</string>
   </property>
  </action>
  <action name="actioncut">
   <property name="text">
    <string>cut</string>
   </property>
  </action>
  <action name="actioncopy">
   <property name="text">
    <string>copy</string>
   </property>
  </action>
  <action name="actionpaste">
   <property name="text">
    <string>paste</string>
   </property>
  </action>
  <action name="actionfind">
   <property name="text">
    <string>find</string>
   </property>
  </action>
  <action name="actionnextfind">
   <property name="text">
    <string>nextfind</string>
   </property>
  </action>
  <action name="actionprevfind">
   <property name="text">
    <string>prevfind</string>
   </property>
  </action>
  <action name="actionfindreplace">
   <property name="text">
    <string>findreplace</string>
   </property>
  </action>
  <action name="actiongo">
   <property name="text">
    <string>go</string>
   </property>
  </action>
  <action name="actionallselect">
   <property name="text">
    <string>allselect</string>
   </property>
  </action>
  <action name="actiondatetime">
   <property name="text">
    <string>datetime</string>
   </property>
  </action>
  <action name="actionwrap">
   <property name="text">
    <string>wrap</string>
   </property>
  </action>
  <action name="actionfont">
   <property name="text">
    <string>font</string>
   </property>
  </action>
  <action name="actionzoomin">
   <property name="text">
    <string>zoomin</string>
   </property>
  </action>
  <action name="actionzoomout">
   <property name="text">
    <string>zoomout</string>
   </property>
  </action>
  <action name="actionzoomrestore">
   <property name="text">
    <string>zoomrestore</string>
   </property>
  </action>
  <action name="actionstatusbarview">
   <property name="text">
    <string>statusbarview</string>
   </property>
  </action>
  <action name="actionhelp">
   <property name="text">
    <string>help</string>
   </property>
  </action>
  <action name="actioninfo">
   <property name="text">
    <string>info</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
